driver_test.go

```diff
@@ -338,3 +309,260 @@ func TestIsDdl(t *testing.T) {
 		}
 	}
 }
+
+func TestExecContextDml(t *testing.T) {
+
+	// Open db.
+	ctx := context.Background()
+	db, err := sql.Open("spanner", dsn)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer db.Close()
+
+	// Set up test table.
+	_, err = db.ExecContext(ctx,
+		`CREATE TABLE TestExecContextDml (
+			key			INT64,
```

Comment:
nit: alignment

Reply:
changed to one tap per line for attributes